Example/EZYVideoPlayer/ViewController.swift

```diff
@@ -7,18 +7,44 @@
 //
 
 import UIKit
+import EZYVideoPlayer
 
 class ViewController: UIViewController {
-
+    
+    @IBOutlet weak var playerView: EZYVideoPlayer!
+    @IBOutlet weak var heightConst: NSLayoutConstraint!
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         // Do any additional setup after loading the view, typically from a nib.
+        playerView.delegate = self
     }
 
     override func didReceiveMemoryWarning() {
         super.didReceiveMemoryWarning()
         // Dispose of any resources that can be recreated.
     }
+}
 
+extension ViewController: EZYVideoPlayerDelegate {
+    
+    func didChangeOrientation(isLandscape: Bool) {
+        view.removeConstraint(heightConst)
+        heightConst = playerView.heightAnchor.constraint(equalTo: view.heightAnchor, multiplier: isLandscape ? 1 : 0.3)
+        view.addConstraint(heightConst)
+    }
+    
+    func didChangedPlayer(status: PlayerStatus) {
+        print("current status => ", status)
+    }
+    
+    func playerDidChanged(position: Float) {
+        print("current position => ", position)
+    }
+    
+    func player(duration: Float) {
+        print("total duration => ", duration)
+    }
```

Comment:
Please remove the print statement if it was for the debugging purpose or you can use the preprocessor macros of swift.

```
#if DEBUG
#endif
```

Reply:
Hey Rama this file won't exist when we install pod as it fall under the example target. So we can safely ignore it. Still thanks for pointing it out.